[
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.5FOArpSQUA"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_number_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_number_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(150): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_string_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_string_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(156): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_text_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_text_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(162): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_number_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_number_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(150): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_string_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_string_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(156): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_text_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_text_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(162): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_number_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:16 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_number_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(150): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_string_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:16 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_string_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(156): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_text_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:16 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_text_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(162): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_number_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:17 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_number_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(150): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_string_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:17 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_string_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(156): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_text_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:17 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_text_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(162): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Declaration of Github_Theme_Upgrader::download_package($package) must be compatible with WP_Upgrader::download_package($package, $check_signatures = false, $hook_extra = []) in /wordpress/wp-content/plugins/theme-updater/assets.php on line 122",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:26 UTC] PHP Fatal error:  Declaration of Github_Theme_Upgrader::download_package($package) must be compatible with WP_Upgrader::download_package($package, $check_signatures = false, $hook_extra = []) in /wordpress/wp-content/plugins/theme-updater/assets.php on line 122\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #5 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"theme-updater\"},\"options\":{\"activate\":true}}) : PHP.run() failed with exit code 255 and the following output: \n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      ... 2 lines matching cause stack trace ...\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: \n        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)\n        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)\n        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)\n        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)\n        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n      response: PHPResponse {\n        httpStatusCode: 500,\n        headers: {\n          'x-powered-by': [ 'PHP/8.0.30-dev' ],\n          'content-type': [ 'text/html; charset=UTF-8' ]\n        },\n        bytes: Uint8Array(0) [],\n        exitCode: 255,\n        errors: ''\n      },\n      source: 'request'\n    }\n  }\n}\n\n",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.f0NeTQ0NOE"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_number_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_number_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(150): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_string_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_string_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(156): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_text_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_text_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(162): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_number_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_number_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(150): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_string_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_string_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(156): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_text_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_text_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(162): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_number_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:16 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_number_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(150): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_string_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:16 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_string_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(156): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_text_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:16 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_text_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(162): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_number_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:17 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_number_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(150): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_string_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:17 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_string_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(156): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_text_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:17 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_text_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(162): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Declaration of Github_Theme_Upgrader::download_package($package) must be compatible with WP_Upgrader::download_package($package, $check_signatures = false, $hook_extra = []) in /wordpress/wp-content/plugins/theme-updater/assets.php on line 122",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:26 UTC] PHP Fatal error:  Declaration of Github_Theme_Upgrader::download_package($package) must be compatible with WP_Upgrader::download_package($package, $check_signatures = false, $hook_extra = []) in /wordpress/wp-content/plugins/theme-updater/assets.php on line 122",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Declaration of Github_Theme_Upgrader::download_package($package) must be compatible with WP_Upgrader::download_package($package, $check_signatures = false, $hook_extra = []) in /wordpress/wp-content/plugins/theme-updater/assets.php on line 122",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:34 UTC] PHP Fatal error:  Declaration of Github_Theme_Upgrader::download_package($package) must be compatible with WP_Upgrader::download_package($package, $check_signatures = false, $hook_extra = []) in /wordpress/wp-content/plugins/theme-updater/assets.php on line 122\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"theme-updater\"},\"options\":{\"activate\":true}}) : PHP.run() failed with exit code 255 and the following output: \n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      ... 2 lines matching cause stack trace ...\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: \n        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)\n        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)\n        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)\n        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)\n        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n      response: PHPResponse {\n        httpStatusCode: 500,\n        headers: {\n          'x-powered-by': [ 'PHP/8.0.30-dev' ],\n          'content-type': [ 'text/html; charset=UTF-8' ]\n        },\n        bytes: Uint8Array(0) [],\n        exitCode: 255,\n        errors: ''\n      },\n      source: 'request'\n    }\n  }\n}\n\n",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.OA5WXjSjHr"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_number_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_number_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(150): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_string_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_string_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(156): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_text_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_text_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(162): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_number_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_number_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(150): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_string_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_string_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(156): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_text_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_text_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(162): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_number_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:16 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_number_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(150): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_string_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:16 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_string_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(156): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_text_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:16 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_text_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(162): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('the-seo-machine...')\n#10 /internal/eval.php(20): activate_plugin('the-seo-machine...')\n#11 {main}</pre>\n for query alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_number_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:17 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_number_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(150): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_number \n            add constraint fk_number_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_string_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:17 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_string_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(156): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_string \n            add constraint fk_string_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: fk_text_to_entity.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:17 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: fk_text_to_entity.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('alter table wp_...')\n#2 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(162): wpdb->get_results('alter table wp_...')\n#3 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(22): TheSeoMachineDatabase->update_if_needed()\n#4 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine-database.php(14): TheSeoMachineDatabase->__construct()\n#5 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(40): TheSeoMachineDatabase::get_instance()\n#6 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(26): TheSeoMachine->__construct()\n#7 /wordpress/wp-content/plugins/the-seo-machine/the-seo-machine.php(99): TheSeoMachine::get_instance()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query alter table wp_the_seo_machine_url_text \n            add constraint fk_text_to_entity foreign key (id_url)\n            references wp_the_seo_machine_url_entity(id)\n            on delete cascade; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/the-seo-machine/the-seo-machine.php'), TheSeoMachine::get_instance, TheSeoMachine->__construct, TheSeoMachineDatabase::get_instance, TheSeoMachineDatabase->__construct, TheSeoMachineDatabase->update_if_needed, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Declaration of Github_Theme_Upgrader::download_package($package) must be compatible with WP_Upgrader::download_package($package, $check_signatures = false, $hook_extra = []) in /wordpress/wp-content/plugins/theme-updater/assets.php on line 122",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:26 UTC] PHP Fatal error:  Declaration of Github_Theme_Upgrader::download_package($package) must be compatible with WP_Upgrader::download_package($package, $check_signatures = false, $hook_extra = []) in /wordpress/wp-content/plugins/theme-updater/assets.php on line 122",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Declaration of Github_Theme_Upgrader::download_package($package) must be compatible with WP_Upgrader::download_package($package, $check_signatures = false, $hook_extra = []) in /wordpress/wp-content/plugins/theme-updater/assets.php on line 122",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:34 UTC] PHP Fatal error:  Declaration of Github_Theme_Upgrader::download_package($package) must be compatible with WP_Upgrader::download_package($package, $check_signatures = false, $hook_extra = []) in /wordpress/wp-content/plugins/theme-updater/assets.php on line 122",
    "log": "/logs/plugins/t/theme-updater/error.json"
  },
  {
    "message": "Declaration of Github_Theme_Upgrader::download_package($package) must be compatible with WP_Upgrader::download_package($package, $check_signatures = false, $hook_extra = []) in /wordpress/wp-content/plugins/theme-updater/assets.php on line 122",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "theme-updater",
    "details": "[24-Apr-2025 19:48:42 UTC] PHP Fatal error:  Declaration of Github_Theme_Upgrader::download_package($package) must be compatible with WP_Upgrader::download_package($package, $check_signatures = false, $hook_extra = []) in /wordpress/wp-content/plugins/theme-updater/assets.php on line 122\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"theme-updater\"},\"options\":{\"activate\":true}}) : PHP.run() failed with exit code 255 and the following output: \n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      ... 2 lines matching cause stack trace ...\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: \n        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)\n        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)\n        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)\n        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)\n        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n      response: PHPResponse {\n        httpStatusCode: 500,\n        headers: {\n          'x-powered-by': [ 'PHP/8.0.30-dev' ],\n          'content-type': [ 'text/html; charset=UTF-8' ]\n        },\n        bytes: Uint8Array(0) [],\n        exitCode: 255,\n        errors: ''\n      },\n      source: 'request'\n    }\n  }\n}\n\n",
    "log": "/logs/plugins/t/theme-updater/error.json"
  }
]
